packages/dashboard-backend/src/devworkspaceClient/services/gettingStartedSamplesApi.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2018-2023 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { IGettingStartedSampleApi } from '../types';
+import * as k8s from '@kubernetes/client-node';
+import { api } from '@eclipse-che/common';
+import { CoreV1API, prepareCoreV1API } from './helpers/prepareCoreV1API';
+import { createError } from './helpers/createError';
+
+const API_ERROR_LABEL = 'CORE_V1_API_ERROR';
+const DEVFILE_METADATA_LABEL_SELECTOR =
+  'app.kubernetes.io/component=getting-started-samples,app.kubernetes.io/part-of=che.eclipse.org';
+const DEFAULT_ICON = {
```

Comment:
@tolusha  Could you move this constant into `packages/dashboard-backend/src/devworkspaceClient/services/helpers/getSampleIcon.ts`?

```
const DEFAULT_ICON = ...

export function getSampleIcon(sample) {
        if (!sample?.icon) {
        return DEFAULT_ICON;
      }
  return sample.icon;
}
```

WDYT?